dataservice.js

```diff
@@ -0,0 +1,47 @@
+let categoriesObject = require('./my-app-react/src/data/categories.json') 
+let productsObject = require('./my-app-react/src/data/products.json')
+
+let combinedProductMap = new Map();
```

Comment:
Have you seen the task description step 8 and 9? Some lines end with `;` and some doesn't. Integrate eslint and fix eslint issues first.